source-code/website/src/renderer/_default.page.client.tsx

```diff
@@ -6,7 +6,6 @@ import { setCurrentPageContext } from "./state.js";
 import type { PageContextRenderer } from "./types.js";
 // only imported client side as web components are not supported server side
 // importing the shoelace components that are used.
-import "@shoelace-style/shoelace/dist/themes/light.css";
```

Comment:
Removed the CSS import here to import the CSS in the stylesheet. The latter allows variable overwrites. Otherwise, the server sends the stylesheet which is then overwritten by the client stylesheet that is imported here.